content/docs/details/scaffolder/index.md

```diff
@@ -0,0 +1,385 @@
+---
+title: Scaffolder
+publishedDate: '2022-05-13T10:53:00.0Z'
+description: Initialize projects and trigger automation from within Backstage. You can prompt parameter inputs and combine scaffolder actions in order to build a scaffolder template to run within Backstage.
+---
+
+# Overview
```

Comment:
Best practice is to only have one `<h1 />` tag per page. The `title` from the YAML metadata is already rendered inside a `<h1 />` on this page. For content, your top level headings should be `<h2 />`.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements#usage_notes

Reply:
will do.

Reply:
done